src/components/Order/Order.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { OrderItem } from "../../components";
+
+export class Order extends React.Component {
+  render() {
+    const { items, total, removeFromOrder } = this.props;
+
+    return (
+      <div className="order">
+        <h1>Order</h1>
+        <ul className="order__items">
+          {items.map((item, i) => {
+            const { name, price, bread, isHot } = item;
+            return (
+              <OrderItem
+                key={i}
+                index={i}
+                name={name}
+                price={price}
+                bread={bread}
+                isHot={isHot}
+                removeFromOrder={removeFromOrder}
+              />
+            );
+          })}
+        </ul>
+        <p className="order__total">
+          TOTAL({items.length}) <span>&pound;{total.toFixed(2)}</span>
```

Comment:
Always wonder if these kind of things like `toFixed` should be done in the return statement or not 🤔 Or if `total` should be passed to `Order` component already formatted

Reply:
It's probably cleaner to do it in 1 place I guess? rather than having a load of `toFixed(2)` littered all over the place